### PR TITLE
Print errors to stdout

### PR DESCRIPTION
**Pull request by varunagrawal**
Fixes for #201

### CODE REVIEW

File: src/pydocstyle/cli.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def run_pydocstyle(use_pep257=False):
 
     count = 0
     for error in errors:
-        sys.stderr.write('%s\n' % error)
+        sys.stdout.write('%s\n' % error)
```
Comment (Nurdok) [inline src/pydocstyle/cli.py]: Now that errors are prints to `stdout`, notice that the test for `--count` in `test_integration.py` needs to be made more robust.

Comment (Nurdok) [inline src/pydocstyle/cli.py]: Instead of checking that the count number is somewhere in the output, check that the it's the last line of `stdout`.

Comment (varunagrawal) [inline src/pydocstyle/cli.py]: @Nurdok done.

**Comment by Nurdok** [comment]
Merged. Thanks!

**Pull request merged**